_docs/022-getting-started-minikube.md

```diff
@@ -55,7 +55,7 @@ For additional details see the minikube [driver installation](https://github.com
 
 ```sh
 minikube start --memory=8192 --cpus=4 \
---kubernetes-version=v1.13.0 \
+--kubernetes-version=v1.13.4 \
```

Comment:
maybe even go for `v1.13.5`

Reply:
how about using v1.14.0 with the latest minikube?